Composer/packages/tools/language-servers/language-generation/src/startSampleServer.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
this file probably belongs to demo, not src, because creating a express instance is not what a library should do